.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: buildpack-deps:trusty
+    working_directory: ~/ask-python
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install Docker client
```

Comment:
need this section
ref: https://circleci.com/docs/2.0/building-docker-images/#example